function/make.go

```diff
@@ -215,6 +223,49 @@ func MakeFunction(name string, function interface{}) MetricFunction {
 			}
 		},
 	}
+
+	for _, option := range options {
+		// TODO: check there aren't multiple conflicting values
+		if option.Name == "WidenBy" || option.Name == "ShiftBy" {
+			sign := -1
+			if option.Name == "ShiftBy" {
+				sign = 1
+			}
+			switch value := option.Value.(type) {
```

Comment:
It's in a switch; there's now a `default` that panics (`MakeFunction` panics whenever something goes wrong, because it's only intended to be called by the programmer as MQE is starting up).